src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -1009,9 +1009,6 @@ static inline int MPIDI_NM_mpi_init_hook(int rank,
     OPA_store_int(&MPIDI_Global.am_inflight_inject_emus, 0);
     OPA_store_int(&MPIDI_Global.am_inflight_rma_send_mrs, 0);
 
-    MPIR_Datatype_init_names();
```

Comment:
Is it okay to remove this?

Reply:
Good question. It worked in testing, but I should confirm the details of why and maybe split into a separate patch.

Reply:
Since `MPIR_Datatype_init_names` also has a static variable escape hatch, I decided to just leave it in to reduce the amount of changes here.

Reply:
👍